contracts/Neumark.sol

```diff
@@ -51,6 +56,7 @@ contract Neumark is
     function issueForEuro(uint256 euroUlps)
         public
         only(ROLE_NEUMARK_ISSUER)
+        acceptAgreement(msg.sender)
```

Comment:
please keep in mind that currently it is called by Commitment smart contract only

Reply:
Yes, but this might change in the future. Jacek recommended to issue the event whenever someone holds NMK, which this function does.